subsys/bluetooth/host/hci_core.c

```diff
@@ -407,6 +407,15 @@ static int hci_le_read_max_data_len(uint16_t *tx_octets, uint16_t *tx_time)
 	*tx_time = sys_le16_to_cpu(rp->max_tx_time);
 	net_buf_unref(rsp);
 
+	if (*tx_octets < BT_HCI_LE_MAX_TX_OCTETS_MIN ||
+	    *tx_octets > BT_HCI_LE_MAX_TX_OCTETS_MAX) {
+		LOG_WRN("tx_octets exceeds the valid range %u", *tx_octets);
+	}
+	if (*tx_time < BT_HCI_LE_MAX_TX_TIME_MIN ||
+	    *tx_time > BT_HCI_LE_MAX_TX_TIME_MAX) {
+		LOG_WRN("tx_time exceeds the valid range %u", *tx_time);
+	}
```

Comment:
Change is OK, but could also consider using the `IN_RANGE` macro, but will give the same result :)

Reply:
`IN_RANGE` looks much much better! Thanks for your comment.